Feedback on page: /guides/v2.4/frontend-dev-guide/css-topics/css_debug.html

# General issue

## Description:

Wrong paragraph 5 
> Add your theme to Grunt configuration. To do this, in the dev/tools/grunt/configs/themes.js file, add your theme to module.exports like following:


## Possible solutions:

You should create dev/tools/grunt/configs/local-themes/themes.js file
or edit
grunt-config.json file

## Additional information:

<!-- (OPTIONAL) What other information can you provide about this issue? -->

Gives and error:
 **$grunt clean**
Loading "Gruntfile.js" tasks...ERROR
>> Error: Cannot find module '.../dev/tools/grunt/configs/local-themes'
Warning: Task "clean" not found. Use --force to continue.

Aborted due to warnings.


<!--
Thank you for taking the time to report this issue!
GitHub Issues should only be created for problems/topics related to this project's codebase.

Before submitting this issue, please make sure you are complying with our Code of Conduct:
https://github.com/magento/devdocs/blob/master/.github/CODE_OF_CONDUCT.md

Issues that do not comply with our Code of Conduct or do not contain enough information may be closed at the maintainers' discretion.

Feel free to remove this section before creating this issue.
-->

--- a/src/guides/v2.3/frontend-dev-guide/css-topics/css_debug.md
+++ b/src/guides/v2.3/frontend-dev-guide/css-topics/css_debug.md
@@ -69,7 +69,6 @@ To compile `.less` files, add your theme to `module.exports` in the Grunt config
 
    **IMPORTANT NOTE** : If grunt gives the error "Error: Cannot find module ....", check the path in your grunt-config.json file and correct as necessary.
 
-
 1. (Optional) If you want to use Grunt for "watching" changes automatically, without reloading pages in a browser each time, install the [LiveReload extension] in your browser.
 
 ## Grunt commands {#grunt_commands}

--- a/src/guides/v2.3/frontend-dev-guide/css-topics/css_debug.md
+++ b/src/guides/v2.3/frontend-dev-guide/css-topics/css_debug.md
@@ -67,6 +67,9 @@ To compile `.less` files, add your theme to `module.exports` in the Grunt config
    -  `<area>`: area code, can be either `frontend` or `adminhtml`.
    -  `<path_to_file>`: path to the root source file, relative to the `app/design/frontend/<Vendor>/<theme>/web` directory. You need to specify all [root source files of the theme]({{ page.baseurl }}/frontend-dev-guide/css-topics/css-preprocess.html#css_preprocess_terms). If your theme [inherits] from a certain theme, and does not contain its own root source files, specify the root source files of the parent theme.
 
+   **IMPORTANT NOTE** : If grunt gives the error "Error: Cannot find module ....", check the path in your grunt-config.json file and correct as necessary.
+
+
 1. (Optional) If you want to use Grunt for "watching" changes automatically, without reloading pages in a browser each time, install the [LiveReload extension] in your browser.
 
 ## Grunt commands {#grunt_commands}

--- a/src/guides/v2.3/frontend-dev-guide/css-topics/css_debug.md
+++ b/src/guides/v2.3/frontend-dev-guide/css-topics/css_debug.md
@@ -67,7 +67,7 @@ To compile `.less` files, add your theme to `module.exports` in the Grunt config
    -  `<area>`: area code, can be either `frontend` or `adminhtml`.
    -  `<path_to_file>`: path to the root source file, relative to the `app/design/frontend/<Vendor>/<theme>/web` directory. You need to specify all [root source files of the theme]({{ page.baseurl }}/frontend-dev-guide/css-topics/css-preprocess.html#css_preprocess_terms). If your theme [inherits] from a certain theme, and does not contain its own root source files, specify the root source files of the parent theme.
 
-   **IMPORTANT NOTE** : If grunt gives the error "Error: Cannot find module ....", check the path in your grunt-config.json file and correct as necessary.
+   **IMPORTANT NOTE** : If grunt gives the error "Error: Cannot find module ....", check the path in your `grunt-config.json` file and correct as necessary.
 
 1. (Optional) If you want to use Grunt for "watching" changes automatically, without reloading pages in a browser each time, install the [LiveReload extension] in your browser.
 

